test/parse.jl

```diff
@@ -1187,3 +1187,14 @@ module Test21607
         x
     end === 1.0
 end
+
+# Basic parsing for in place assign
+@test parse("a@b = 1") == :(a@b = 1)
```

Comment:
Primarily I wanted to make sure it doesn't throw, because I'm not sure the parsing is final yet. I also find it useful, because that way I can use parse.jl to test JuliaParser.